Header.css

```diff
@@ -0,0 +1,33 @@
+@charset "utf-8";
```

Comment:
Instead of the multiple CSS files, you could always put the elements you wanted to edit in the same class, but give them different IDs on each HTML page to refer to them. On your restaurant.html, you could do something like...

`<body id = "restaurant">
     <div class = "mainDiv">`

and then do something similar on the index..

`<body id = "home">
     <div class = "mainDiv">`

and then finally, refer back to them on a single CSS..

```css
#restaurant .mainDiv {
border: 1px solid green;
width: 200px;
}

#home .mainDiv {
border: 1px solid red;
width: 100px;
}